apps/audits/utils.py

```diff
@@ -48,7 +49,10 @@ def _get_instance_field_value(
             if getattr(f, 'attname', None) in model_need_continue_fields:
                 continue
 
-            value = getattr(instance, f.name, None) or getattr(instance, f.attname, None)
+            try:
+                value = getattr(instance, f.name, None) or getattr(instance, f.attname, None)
+            except ObjectDoesNotExist:
+                continue
             if not isinstance(value, (bool, int)) and not value:
                 continue
 
```

Comment:
There are no major issues found with the code. The changes suggested by validation of `value` when assigning default values seem to be appropriate for this use case. No other obvious errors were detected during inspection.

If there is anything else you would like me to do regarding coding best practices, please let me know!